Makefile

```diff
@@ -579,9 +579,15 @@ contracts/test/prover/proofs/%.json: $(arbitrator_cases)/%.wasm $(prover_bin)
 	@touch $@
 
 .make/cbrotli-wasm: $(DEP_PREDICATE) $(ORDER_ONLY_PREDICATE) .make
-	test -f target/lib-wasm/libbrotlicommon-static.a || ./scripts/build-brotli.sh -w 
-	test -f target/lib-wasm/libbrotlienc-static.a || ./scripts/build-brotli.sh -w 
-	test -f target/lib-wasm/libbrotlidec-static.a || ./scripts/build-brotli.sh -w 
+ifdef NODOCKERINDOCKER
+	test -f target/lib-wasm/libbrotlicommon-static.a || ./scripts/build-brotli.sh -w
+	test -f target/lib-wasm/libbrotlienc-static.a || ./scripts/build-brotli.sh -w
+	test -f target/lib-wasm/libbrotlidec-static.a || ./scripts/build-brotli.sh -w
+else
+	test -f target/lib-wasm/libbrotlicommon-static.a || ./scripts/build-brotli.sh -w -d
```

Comment:
I'd try setting a variable to `-d` or empty and use that here in order to avoid duplicating all of these